crates/nu-std/testing.nu

```diff
@@ -287,6 +287,8 @@ export def run-tests [
     --list,                               # list the selected tests without running them.
     --threads: int@"nu-complete threads", # Amount of threads to use for parallel execution. Default: All threads are utilized
 ] {
+    use std log
+
```

Comment:
i don't see any use of the `log` commands in this command :thinking:

```suggestion
```

Reply:
Good catch - Some other commands in there broke because `use std` was gone, so I had pasted it into each of them; not noticing that one didn't need it.

Removed it in the pending commit.